src/CitizenAction/CitizenActionParticipantsExporter.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace AppBundle\CitizenAction;
+
+class CitizenActionParticipantsExporter
+{
+    public function export(array $participants): string
+    {
+        $handle = fopen('php://memory', 'r+');
+        fputs($handle, chr(0xEF).chr(0xBB).chr(0xBF)); // add BOM to fix UTF-8 in Excel
+        fputcsv($handle, ['N° d\'enregistrement', 'Prénom', 'Nom', 'Âge', 'Ville', 'Date d\'inscription']);
+
+        foreach ($participants as $participant) {
+            if (!array_key_exists('uuid', $participant)
+                || !array_key_exists('firstName', $participant)
+                || !array_key_exists('lastName', $participant)
+                || !array_key_exists('age', $participant)
+                || !array_key_exists('cityName', $participant)
+                || !array_key_exists('createdAt', $participant)
+                || !$participant['createdAt'] instanceof \DateTime
+            ) {
+                throw new \InvalidArgumentException('Invalid participant object given');
```

Comment:
This error message should indicate which keys are missing.
I think we could use `\Symfony\Component\OptionsResolver\OptionsResolver` to help us validating these required keys and more.
If we want to stay with something simple we can always use the `array` functions to do something similar.